admin-api-server/script/entrypoint

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
```

Comment:
To stay consistent with other scripts, and since we dont want to do anything related to the PWD of where this script is called ever, best to do a `cd $SCRIPT_DIR/..` here.

And replace `$SCRIPT_DIR/` below with `script/`.

Anyways at the moment there is a mistake because the final statement `node dist/src/main.js` doesn't take this $SCRIPT_DIR into account. Adding the `cd $SCRIPT_DIR/..` will fix that as well.

Reply:
This script will be used only in docker and always as entrypoint. It is not necessary to use this variable. Just need to use `script/..` in two script work better. In `migration` script we already use relational path ( of app folder ), therefore using `cd script` may cause error.

It is now simply `script/...` used in two script/db call.

Reply:
Agree. Fixed now